styles/Canonical/001-English-words-spelling-suggestions.yml

```diff
@@ -96,4 +96,5 @@ swap:
   'virtualization': virtualisation
   'whiskey': whisky
   'woolen': woollen
-  'yogurt': yoghurt
\ No newline at end of file
+  'yogurt': yoghurt
```

Comment:
Hi @evilnick ! Sorry, I am curious about this change.
Do the rules need that last blank line?
thank you!

Reply:
the rules don't but plenty of linters (including the built-in GH one) get upset if you don't end every file with a \n

Reply:
Ah, ok, thank you.